firewall/iptables/iptables.go

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright (C) 2018 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package iptables
+
+import (
+	"bufio"
+	"bytes"
+	"os/exec"
+	"strings"
+
+	log "github.com/cihub/seelog"
+	"github.com/mysteriumnetwork/node/firewall"
+	"github.com/pkg/errors"
+)
+
+const (
+	logPrefix = "[iptables] "
+
+	outputChain     = "OUTPUT"
+	killswitchChain = "CONSUMER_KILL_SWITCH"
+
+	addChain         = "-N"
+	appendRule       = "-A"
+	insertRule       = "-I"
+	listRules        = "-S"
+	removeRule       = "-D"
+	removeChainRules = "-F"
+	removeChain      = "-X"
+
+	jumpTo        = "-j"
+	sourceIP      = "-s"
+	destinationIP = "-d"
+	module        = "-m"
+
+	protocol        = "-p"
+	tcp             = "tcp"
+	udp             = "udp"
+	destinationPort = "--dport"
+
+	conntrack    = "conntrack"
+	ct_state     = "--ctstate"
+	ct_state_new = "NEW"
+
+	reject = "REJECT"
+	accept = "ACCEPT"
+
+	version = "--version"
+)
+
+var iptablesExec = func(args ...string) ([]string, error) {
```

Comment:
Why not to use just named function? 
```suggestion
func iptablesExec(args ...string) ([]string, error) {
```

Reply:
To be able to replace it in Tests with mocked one.